_data/education.yml

```diff
@@ -127,6 +127,16 @@ websites:
       software: Yes
       doc: https://www.ecu.edu/cs-itcs/office365/multi-factor-authentication.cfm
 
+    - name: Eastern Michigan University
+      url: https://www.emich.edu
+      img: emu.png
+      tfa: Yes
+      hardware: Yes
+      software: Yes
+      exceptions:
+          text: "Available for university employees, not students."
```

Comment:
probably best to mirror their language (since student employees are actually valid)?

what about `"Available for employees"`

Reply:
@programmer94 just a friendly ping to get this top of inbox for you. Sounds like this PR is good to go with just a few last tweaks